tests/runtest.proj

```diff
@@ -121,22 +126,6 @@ $(_XunitEpilog)
 </Project>
         ]]>
       </XunitWrapperGenCsProj>
-      <XunitWrapperGenPackConfig>
-        <![CDATA[
-{
-  "dependencies": {
```

Comment:
True, I could do that. I think the main downside is that if I generate it during runtest, I would have to restore it during runtest too instead of using the main batch restore in buildtest. (Hitting the startup cost of dotnet restore.) I could generate the project.json during buildtest to get it in before batch restore, but I don't think there's any code gen during that step so it seems like something to avoid. I think I'll stick with the project.json in git.

I don't think this will end up causing much/any noise in PRs though, since this project.json only changes when xunit is upgraded which as far as I can tell is extremely rare.